provisions/roles/nginx/templates/registry.centos.org.conf.j2

```diff
@@ -12,18 +12,21 @@ upstream registry {
   server 127.0.0.1:5000 max_fails=0;
 }
 
+upstream registry-ui {
+  server 127.0.0.1:9000 max_fails=0;
+}
```

Comment:
how about adding /Home here? so that user does not need to click on the browse repositories